sdk/core/azure-core-management/src/main/java/com/azure/core/management/implementation/LocationPollStrategy.java

```diff
@@ -84,24 +84,24 @@ public HttpRequest createPollRequest() {
 
     @Override
     public Mono<HttpResponse> updateFromAsync(HttpResponse httpPollResponse) {
-        return ensureExpectedStatus(httpPollResponse, new int[] {202})
-                .map(response -> {
-                    final int httpStatusCode = response.statusCode();
-                    updateDelayInMillisecondsFrom(response);
-                    if (httpStatusCode == 202) {
-                        String newLocationUrl = getHeader(response);
-                        if (newLocationUrl != null) {
-                            try {
-                                data.locationUrl = new URL(newLocationUrl);
-                            } catch (MalformedURLException mfue) {
-                                throw logger.logExceptionAsError(Exceptions.propagate(mfue));
-                            }
+        return ensureExpectedStatus(httpPollResponse, new int[]{202})
+            .map(response -> {
```

Comment:
Does this break checkstyles? I remember 8 space continuation being a rule, but intellij reformats it in a weird way.